modules/vaos/app/services/vaos/cc_supported_sites_service.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require_relative '../vaos/concerns/headers'
+
+module VAOS
+  class CCSupportedSitesService < Common::Client::Base
+    include Common::Client::Monitoring
+    include SentryLogging
+    include VAOS::Headers
+
+    configuration VAOS::Configuration
+
+    STATSD_KEY_PREFIX = 'api.vaos'
+
+    attr_reader :user
+
+    def initialize(user)
+      @user = user
+    end
+
+    def get_supported_sites(site_codes)
+      with_monitoring do
+        response = perform(:get, url(site_codes), nil, headers(user))
+        {
+          data: deserialize(response.body),
+          meta: {}
+        }
+      end
+    end
+
+    private
+
+    def deserialize(json_hash)
+      json_hash[:sites_supporting_var].map do |request|
+        OpenStruct.new(request)
+      end
+    rescue => e
+      log_message_to_sentry(e.message, :warn, invalid_json: json_hash)
+      []
```

Comment:
Only concern here is that we miss out on automated error monitoring/metrics by catching the error. Is it likely that we'd get a 200 response with malformed JSON? IMO if it's ok that CC data is blank the FE could handle the error gracefully while still allowing the backend to raise an error (like a 422)

Reply:
I think this was a cut and paste, and yeah -- i think I agree with you on this! Lets tackle this across all the services that are doing this as part of a refactor. And maybe we want to prioritize that to make sure we catch anything with it sooner rather than later.